Can't install on npm 3+

```
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "i" "portastic" "--save"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.4
npm ERR! path /code/session-server/node_modules/portastic/bin/portastic
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/code/session-server/node_modules/portastic/bin/portastic'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
```

Hi! I get the same error with npm v2.14.4.

```
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "portastic"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! path C:\Users\**\**\**\**\node_modules\portastic\bin\portastic
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod 'C:\Users\**\**\**\**\node_modules\portastic\bin\portastic'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
´´´
```

--- a/.npmignore
+++ b/.npmignore
@@ -1,6 +1,6 @@
 **/*
+!bin/**/*
 !lib/**/*
-!bin/targz
 !package.json
 !index.js
 !LICENSE

--- a/bin/portastic
+++ b/bin/portastic
@@ -1,4 +1,4 @@
-#!/usr/bin/node
+#!/usr/bin/env node
 
 var portastic = require('../');
 var commander = require('commander');

